azure-pipelines.yml

```diff
@@ -0,0 +1,51 @@
+# Microsoft JDBC Driver for SQL Server CI Build triggers tests against below SQL Servers:
+#   - SQL Server 2017
+#   - SQL Server 2008 R2
+jobs:
+- job: "CI_Build"
+  pool:
+    name: 'JDBC Build Pool'
+    demands: maven
+  strategy:
+    matrix:
+      SQL-2017:
+        Target_SQL: 'SQL-2k17-03'
+      SQL-2008R2:
+        Target_SQL: 'SQL-2k8R2-SP3-1'
+    maxParallel: 2
+  steps:
+  - powershell: |
+      mkdir AE_Certificates
+      cd AE_Certificates
+      $cert = New-SelfSignedCertificate -dns "AlwaysEncryptedCert" -CertStoreLocation Cert:CurrentUser\My
+      $pwd = ConvertTo-SecureString -String "password" -Force -AsPlainText
```

Comment:
Will we have issues including the password here and below.

Reply:
Secured